### ISSUE TITLE
Create a collection of open-access 3D resources for demo usage

### ISSUE THREAD

**Issue by JulieWinchester** [IIIF/3d#27]
For creating interoperable demos of IIIF 3D progress, we need a collection of open-access (CC0 or other permissively licensed) 3D models. Participants should provide links to possible models for this purpose **with a statement of specific licensing (with sourcing, if possible) applied to the models**. Once we gather a good collection, we can rehost copies of these models in a central repository for use in demos.

**Comment by JulieWinchester** [comment]
Some resources from the [Smithsonian's 3D CC0 resources page](https://3d.si.edu/cc0). All models are in GLB format (without DRACO compression), have coordinate units with meter-scale, and are lowest-resolution available from the site. I've tried to mostly include files <5MB in file size. 

-  Saxophone (2.6 MB), media file specifically labeled as CC0 on resource page. [Resource page](https://3d.si.edu/object/3d/alto-saxophone-owned-and-played-charlie-parker:c7c58ff8-fd1f-4cf6-9813-6bca3cd0b8b3), [Model URL](https://3d-api.si.edu/content/document/3d_package:c7c58ff8-fd1f-4cf6-9813-6bca3cd0b8b3/resources/2019-10-1a-g_001-100k-2048_std.glb)
- Pilot Whale skull (3.1 MB), media file specifically labeled as CC0 on resource page. [Resource page](https://3d.si.edu/object/3d/globicephala-melas-melas-traill-1809-skull-and-mandible:a8ab02ba-bcae-470a-9df4-db1265b25905), [Model URL](https://3d-api.si.edu/content/document/3d_package:a8ab02ba-bcae-470a-9df4-db1265b25905/resources/USNM_VZ_20957_pilot_whale_skull-100k-2048_std.glb)
- Fossil whale excavation site model (24.3 MB), media file specifically labeled as CC0 on resource page. This model is large, but it's included for its ability to be combined with the whale skull for multi-model demos. [Resource page](https://3d.si.edu/object/3d/fossil-whale-excavation-site-mpc-684:64ff5dbe-cf46-486e-8b3a-cb8e1fd7895c), [Model URL](https://3d-api.si.edu/content/document/3d_package:64ff5dbe-cf46-486e-8b3a-cb8e1fd7895c/resources/mpc_684-100k-web.glb)
- Porcelain figure of a dancer (1.2 MB), media file specifically labeled as CC0 on resource page. [Resource page](https://3d.si.edu/object/3d/figure-dancer:88de08dd-b8ab-470a-b987-ed6fe35def04), [Model URL](https://3d-api.si.edu/content/document/3d_package:88de08dd-b8ab-470a-b987-ed6fe35def04/resources/figure-of-a-dancer-150k-2048-medium-100k-2048_std_draco.glb)
- Heart monitor prototype (1.6 MB), media file specifically labeled as CC0 on resource page. [Resource page](https://3d.si.edu/object/3d/prototype-digital-heart-rhythm-monitor-developed-dr-l-julian-haywood:f2d0d103-24d5-4ec5-a768-b983873ea1f1), [Model URL](https://3d-api.si.edu/content/document/3d_package:f2d0d103-24d5-4ec5-a768-b983873ea1f1/resources/PrototypeHeartRateMonitor_2017-42-100k-2048_std_draco.glb)

**Comment by JulieWinchester** [comment]
3D IIIF manifest test/example app https://github.com/mikeapp/iiif-threejs uses the two following models, both from Sketchfab and licensed with CC0:

* Bust of Augustus, media file specifically labeled as CC0 on resource page. [Resource page](https://sketchfab.com/3d-models/buste-dauguste-couronne-de-chene-6ccc01ad1cda47f3b2511ef6fbf07e0b), [Model URL](https://github.com/mikeapp/iiif-threejs/raw/main/public/models/augustus.glb)
* Venus de Milo, media file specifically labeled as CCO on resource page. [Resource page](https://sketchfab.com/3d-models/venus-de-milo-statuestexturingchallenge-smk-2983d92ac4e744f485492580ca7629f2), [Model URL](https://github.com/mikeapp/iiif-threejs/raw/main/public/models/venus.glb)

**Comment by Cook4986** [comment]
- Cela | Ruins of a nuns' cell 
[Resource page](https://sketchfab.com/3d-models/cela-ruins-of-a-nuns-cell-507935f6790e4c278c0cd723d5dd938c)
Internal camera (but relatively high poly)

**Comment by edsilv** [comment]
Here are the example models used by Google model-viewer.

Sketchfab: https://github.com/google/model-viewer/tree/master/packages/modelviewer.dev/assets/SketchfabModels

"Thor and the Midgard Serpent" (https://skfb.ly/69X8V) by Mr. The Rich is licensed under Creative Commons Attribution (http://creativecommons.org/licenses/by/4.0/)

Shopify: https://github.com/google/model-viewer/tree/master/packages/modelviewer.dev/assets/ShopifyModels

General: https://github.com/google/model-viewer/tree/master/packages/shared-assets/models

Not sure what license these have, but the root [LICENSE](https://github.com/google/model-viewer/blob/master/LICENSE) file in the project is Apache 2.

**Comment by JulieWinchester** [comment]
Thanks all! From all the options given thus far, here are my nominations for 6 models that I might suggest for the final list. We will discuss this during the meeting today, happy to strike any of these or add others.

* Astronaut from Google model viewer assets (2.6 MB): This has already become commonly used by our group and it's a small simple model. Seems ideal for our purposes. 
* Venus de Milo model from SMK's Sketchfab (3 MB): This is an instantly recognizable piece and is a super small mesh with texture to boot. 
* Saxophone from Smithsonian collections (2.6 MB): It would be helpful to have an object prop to pair with roughly humanoid-shaped models. The original page for this model also has point label annotations, so it's a good use case for those as well.
* Pilot whale skull from Smithsonian collections (926 KB): I think it's good to have some representation from biological collections, since they are quite commonly 3D scanned at this point. Also, for multi-object manifests, this pairs well with the Pilot whale mandible below.
* Pilot whale mandible from Smithsonian collections (1.2 MB): See above, it's biological material and it pairs perfectly with the skull for a real-world use case.
* Ruins of a Nun's Cell (109 MB): This is quite a bit larger than the other models by file size, but I think this is a model unlike any of the others, and it comes with annotations on the Sketchfab page. Also, just looking at the triangle count on the model, it may be possible to shrink this down some by compressing the 2D texture images further? But even if it's 100MB, I think this is worth including for the uniqueness of type and the presence of annotations.

**Comment by JulieWinchester** [comment]
During the 3D TSG meeting on 9/20, there was general consensus agreement with this list.

**Comment by JulieWinchester** [comment]
We've received confirmation of licensing and permission to use the Venus de Milo, the saxophone, the pilot whale skull components, and the nun's cell models. I've added these to the repository in `assets/`. 

Communicating with the model viewer folks has indicated that the low poly Astronaut as well as the two Shopify assets are not owned by model viewer but are being used by model viewer with attribution licenses like CC-BY. (Incidentally, the low poly Astronaut looks like it initially came from Google Poly, which having now shut down means the original source for this model is kaput.) We may want to find other low-poly models to add to this collection, so that we can stick with our CC0/fully-permissive criterion.

**Comment by JulieWinchester** [comment]
After much discussion in the TSG meetings, we have decided to make an exception to the principle of only including open access models to allow for the inclusion of the low poly astronaut model. The model has been added to the repository under `assets/astronaut` and full attribution has been made there.

I am closing this issue as done for now, but we can always revisit adding more models to our asset collection later.
